### ISSUE TITLE
error[E0277]: the trait bound `iced_wgpu::renderer::Renderer: iced_native::widget::image::Renderer` is not satisfied

### ISSUE THREAD

**Issue by LeCyberDucky** [hecrj/iced#258]
Hi, 

I'm trying to compile the "svg" example using the newest release. With the "0.1.0-beta" release, it works as expected. With the new release, however, I run into the following error:

> error[E0277]: the trait bound `iced_wgpu::renderer::Renderer: iced_native::widget::svg::Renderer` is not satisfied

with the explanation

> the trait `iced_native::widget::svg::Renderer` is not implemented for `iced_wgpu::renderer::
> Renderer`

I hope we can work this out. Please tell me if more information is needed :)

**Comment by hecrj** [comment]
How are you exactly compiling it? `cargo run --package svg` works for me.

If you copied the example into your own project, you may need to enable [the `svg` feature](https://github.com/hecrj/iced/blob/master/Cargo.toml#L18) (I should document this!).

**Comment by LeCyberDucky** [comment]
Ah, yes. Enabling the svg feature did the trick. Thanks for the quick help!

**Comment by ardijanr** [comment]
New guy here, how exactly did you enable this?

I tried:

`[features]
svg = ["iced_wgpu/svg"]

[dependencies]
iced = "0.2"
iced_wgpu = "0.3"
`

**Comment by LeCyberDucky** [comment]
The example directories contain TOML files that you can use as a reference. But for SVG, it should be something like this: 

`iced = { version = "0.2.0", features = ["svg"] }`

I hope that helps :)

**Comment by ardijanr** [comment]
Yeah, that did the trick. Found that it also worked when compiling and running with:

cargo run --features "svg"

But your solution is way better, kinda new to rust. Thanks alot for the help and fast reply.
